cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -1477,9 +1479,9 @@ void reader::impl::setup_next_subpass(read_mode mode)
 
 void reader::impl::create_global_chunk_info()
 {
-  auto const num_rows         = _file_itm_data.global_num_rows;
-  auto const& row_groups_info = _file_itm_data.row_groups;
-  auto& chunks                = _file_itm_data.chunks;
+  auto const num_rows   = _file_itm_data.global_num_rows;
+  auto& row_groups_info = _file_itm_data.row_groups;
```

Comment:
Removed const as the first row group's start_row needs to be adjusted for skip_rows